site/content/docs/monitoring/monitoring-local-environments.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Monitoring local environments
+weight: 7
+menu:
+  docs:
+    parent: "Monitoring"
+---
+
+Checkly allows you to check your publicly accessible websites and APIs. However, by default Checkly won't have access to sites that aren't publicly available (for instance localhost). 
+
+With [ngrok](https://ngrok.com/) you can secure a tunnel to your localhost, so you can run checks on your internal sites directly, without the need of deploying every change.
+
+Here 
+
+1. Go to [https://ngrok.com/](https://ngrok.com/) and create an account. 
+2. Download Ngrok, install it and get it started [https://dashboard.ngrok.com/get-started/setup](https://dashboard.ngrok.com/get-started/setup)
+3. Say you want to run checks on [http://localhost:80](http://localhost:80), you'll need expose port 80, with the command `./ngrok http 80`, or `ngrok http 80` if you extracted ngrok on your system's $PATH.
+4. You'll get a screen similar to this.
+![ngrok](/docs/images/monitoring/ngrok.png)
+5. Then you can use the "forwarding" URL in Checkly.
+![script](/docs/images/monitoring/script.png)
+
+6. The check will reach successfully your localhost and ngrok will show successful HTTP requests.
+![ngrok200](/docs/images/monitoring/ngrok200.png)
```

Comment:
Please review all the alt texts, such as "ngrok200" and change them to meaningful and concise descriptions of the image. Here is a good guide: https://supercooldesign.co.uk/blog/how-to-write-good-alt-text